generator/src/test/java/org/stjs/generator/writer/namespace/NamespaceGeneratorTest.java

```diff
@@ -111,10 +111,7 @@ public void testNamespaceDefinedInConfiguration() {
 
 		assertCodeContains(Namespace13_generator_configuration.class, "" +
 						"stjs.ns(\"my.own.namespace\");\n" +
-						"my.own.namespace.Namespace13_generator_configuration = function(outerClass$0) {\n" +
-						"    this._outerClass$0 = outerClass$0;\n" +
-						"};\n" +
-						"my.own.namespace.Namespace13_generator_configuration = stjs.extend(my.own.namespace.Namespace13_generator_configuration, null, [], function(constructor, prototype) {",
+						"my.own.namespace.Namespace13_generator_configuration = function() {}",

```

Comment:
There was a bug: this class has no `outerClass$0`.

I also simply removed the part because this is not what this specific test case was intended for:
```
my.own.namespace.Namespace13_generator_configuration = stjs.extend(my.own.namespace.Namespace13_generator_configuration, null, [], function(constructor, prototype) {
```

If you look at the test class, the important part for this test case is:
```
		assertCodeContains(Namespace13_generator_configuration.class, "" +
						"var date = new java_util_custom_namespace.Date()._constructor();",
				generatorConfigurationBuilder.build());
```